packages/protocol/proto/sdk/session.proto

```diff
@@ -264,13 +295,20 @@ message ServerEvent {
             shared.Kind kind = 3;
         }
 
+        message ChannelMessage {
+            string key = 1;
```

Comment:
What is the meaning of key field?

Reply:
the key is the channel key, which can be consider a channel's identifier key